src/Network/Minio/S3API.hs

```diff
@@ -1,5 +1,7 @@
+{-# LANGUAGE TupleSections #-}
```

Comment:
This can be removed. I know it was added to get stylish-haskell to work, but it can be removed once the formatting worked. This is being fixed properly in https://github.com/minio/minio-hs/pull/83 as well - we will add a config file for stylish-haskell that includes all the lang extensions for the project.